tensorflow_serving/servables/tensorflow/session_bundle_config.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 
+import "tensorflow/core/protobuf/config.proto";
```

Comment:
Sure. Sorry. My bad. This is indeed wrong order for both oss and internal trees.